RELEASE_NOTES.md

```diff
@@ -5,6 +5,8 @@ Changes:
   * Changed Edit Study page to show validation messages instead of error page when saving fails
   * Changed Edit Instrument page to show validation messages instead of error page when saving
     fails
+  * Changed Edit Submission page to show validation messages instead of error page when saving
+    fails
```

Comment:
Maybe these could just be one long Change? eg 'Changed Edit Submission, Instrument, Study, [...] to show validation messages instead of error page when saving fails'

Reply:
Yeah, would be better that way, though it makes rebasing a little more painful. I'm lead next week and will condense during release